workflow/rocoto/gfs_tasks.py

```diff
@@ -1175,9 +1175,17 @@ def wavepostbndpnt(self):
         return task
 
     def wavepostbndpntbll(self):
+
+        # The wavepostbndpntbll job runs on forecast hours up to 180 hours
+        last_fhr = Tasks._get_forecast_hours(self.cdump,
+                                             self._configs['wavepostbndpntbll'],
+                                             component='atmos')[-1]
+        if last_fhr > 180:
+            last_fhr = 180
```

Comment:
Same as above.

Reply:
Fixed.

Reply:
`FHMAX_WAV_IBP` is already clipped to `FHMAX`, so you shouldn't need to do it again here.